src/Catty/PocketPaint/ViewController/PaintViewController.m

```diff
@@ -907,17 +926,19 @@ - (void)saveAction
             } else {
                 dispatch_async(dispatch_get_main_queue(), ^{
                     [[[[[AlertControllerBuilder alertWithTitle:nil message:kLocalizedNoAccesToImagesCheckSettingsDescription]
-                     addCancelActionWithTitle:kLocalizedCancel handler:nil]
-                     addDefaultActionWithTitle:kLocalizedSettings handler:^{
-                         if ([self.delegate respondsToSelector:@selector(addPaintedImage:andPath:)]) {
-                             if (self.editingPath) {
-                                 [self.delegate addPaintedImage:self.saveView.image andPath:self.editingPath];
-                             } else {
-                                 [self.delegate addPaintedImage:self.saveView.image andPath:@"settings"];
-                             }
-                         }
-                         [[UIApplication sharedApplication] openURL:[NSURL URLWithString:UIApplicationOpenSettingsURLString] options:@{} completionHandler:nil];
-                     }] build]
+
+                        addCancelActionWithTitle:kLocalizedCancel handler:nil]
+                       addDefaultActionWithTitle:kLocalizedSettings handler:^{
+                        if ([self.delegate respondsToSelector:@selector(addPaintedImage:andPath:)]) {
+                            if (self.editingPath) {
+                                [self.delegate addPaintedImage:self.saveView.image andPath:self.editingPath];
+                            } else {
+                                [self.delegate addPaintedImage:self.saveView.image andPath:@"settings"];
+                            }
+                        }
+                        [[UIApplication sharedApplication] openURL:[NSURL URLWithString:UIApplicationOpenSettingsURLString]];
```

Comment:
This line gives a compiler warning:
`'openURL:' is deprecated: first deprecated in iOS 10.0`

Please bring back the initial `openURL` call (with `options` and `completionHandler` paramaters).